tests/vertex_map_tests.cc

```diff
@@ -25,15 +33,6 @@ limitations under the License.
 #include <utility>
 #include <vector>
 
-#include <grape/fragment/immutable_edgecut_fragment.h>
-#include <grape/fragment/loader.h>
-#include <grape/fragment/mutable_edgecut_fragment.h>
-#include <grape/fragment/partitioner.h>
-#include <grape/grape.h>
-#include <grape/util.h>
-#include <grape/vertex_map/global_vertex_map.h>
-#include <grape/vertex_map/local_vertex_map.h>
-
```

Comment:
Don't touch this part, the self headers should be placed after std headers, the original behaviour should be ok.

Reply:
It's done by clang-format automatically, in other files of this repo, the include order is same with the changed by me, you can see this file [run_app.h](https://github.com/alibaba/libgrape-lite/blob/master/examples/analytical_apps/run_app.h).
Or, I can change the <> to ""

Reply:
Ok. We could get this merged and I will tweak the clang-format configuration.

Thanks!